docs/running-node-aws-terraform-ansible/deployment.md

```diff
@@ -0,0 +1,184 @@
+
+```hcl
+private_sentries_config = {
+    enable        = true
+    nodes_count   = 2
+    instance_type = "t3.medium"
+}
+```
+- Private sentry nodes are created in the region as Validator by default
+- Can be disabled by setting `enable = false`
+- Only one instance of private sentry is created with static ip address
+
+### Public Sentries (optional):
+```hcl
+public_sentries_config = {
+    enable        = true
+    enable_ipv6   = false
+    nodes_count   = 2
+    instance_type = "t3.medium"
+
+    regions = [
+        1,
+        2
+    ]
+}
+```
+- Requires `Private Sentries` being enabled
+- Can be configured to have IPv6 addresses using `enable_ipv6 = true`
+    > **_Note:_** Number of available IPv4 static addresses is restricted to 5 per region on AWS by default
+- Can be configured to run on multiple regions
+
+### Observers (optional):
+```hcl
+observers_config = {
+    enable           = true
+    nodes_count      = 3
+    instance_type    = "t3.medium"
+    root_domain_name = "matterprotocol.com"
+    enable_tls       = true
+
+    regions = [
+        1,
+        2
+    ]
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When `root_domain_name` parameter is set, will be serving requests under `on.<root_domain_name>` domain
+    - Name servers for `root_domain_name` servers must be pointed to AWS and managed by the AWS account
+- TLS can be enabled or disabled
+- Can be configured to run on multiple regions
+
+### Prometheus (optional)
+```hcl
+prometheus_config = {
+  enable        = true
+  instance_type = "t3.small"
+}
+```
+
+- Requires `Private Sentries` being enabled
+- When enabled runs a dedicated Prometheus server on Private Sentries VPC to collect Tendermint metrics from all DCL nodes
+- Collected metrics are written to AWS [AMP workspace](https://aws.amazon.com/prometheus/)
+
+### 2. Configure Ansible inventory variables in [`deployment/ansible/inventory/aws/group_vars/all.yaml`]
+```yaml
+chain_id: test-net2
+
+dcl_home: /var/lib/dcl/.dcl
+dcl_version: 0.9.0
+dcld:
+  version: "{{ dcl_version }}"
+  path: "{{ dcl_home }}/cosmovisor/genesis/bin/dcld"
+cosmovisor:
+  version: "{{ dcl_version }}"
+  user: cosmovisor
+  group: dcl
+  path: /usr/bin/cosmovisor
+  home: "{{ dcl_home | dirname }}"
+
+dcld_checksums:
+  0.9.0: c333d828a124e527dd7a9c0170f77d61ad07091d9f6cd61dd0175a36b55aadce
+cosmovisor_checksums:
+  0.9.0: c05705efe5369b9d83e65ef7b252bd7c610eec414ae3f6c08681bcf49dc38e6d
+
+dcld_download_url: "https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v{{ dcld.version }}/dcld"
+dcld_binary_checksum: "sha256:{{ dcld_checksums[dcld.version] }}"
+cosmovisor_download_url: "https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v{{ cosmovisor.version }}/cosmovisor"
+cosmovisor_binary_checksum: "sha256:{{ cosmovisor_checksums[cosmovisor.version] }}"
+```
+- Specify DCL network `chain_id`
+- Specify `dcld` and `cosmovisor` versions and checksums
+
```

Comment:
It's not clear from the instructions how to join an existing pool?
How to enable state-sync? How to setup genesis and persistent_peers?